### DISCUSSION TITLE
Moving from Raspberry Docker to Synology Docker

### DISCUSSION THREAD

**Discussion by dansche** [teslamate-org/teslamate#3287] [General]
Hi experienced users

As a noob enjoying since 46k KM the Teslamate and have recorded since than everything I want to move from a Raspberry Pi 4 to a Synology NAS, I've installed the new instance to Synology and stopped where I create the Key. Is there someone has done that allready? So if I generate the key as far as I know the Software on Synology receives than the Data from my car. On the raspberry I am able to make the Backup file as I did allways before Updating, but how can I restore all the data to the new instance and from than on record on the Synology?

Please be patient as I'm not an expert so if you describe, make it easy ;-)

Thanks

**Comment by cwanja** [comment]
Simply follow the restore options on the documents. https://docs.teslamate.org/docs/maintenance/backup_restore

**Reply by dansche** [reply]
I tried to follow those, but if you intall it on Synology there is no YML file in the file system, you do that all over the GUI. If i'm in the Teslamate folder, there are only the following directories...
datagrafana
mosqconf
mosqdata
mosqlogs
db / which is not accessible

So the command "docker-compose stop teslamate" in the teslamate folder bring the error "no configuration file provided: not found"

I have made the backup file from the previous instance "teslamate.bck" this file is 1.1G big, have that copied away from raspberry to synology in the teslamate folder... but there I'm stuck...

Help very much appreciated

**Reply by av642** [reply]
Hi - were you able to solve this?  This is where I'm having an issue.  teslamate.bck is already in the teslamate folder.  How to update is where I'm lost. 

Currently using Container manager and an updated YAML file. 

Have tried SSH via putty and root access.  When putting in terms - synology doesn't recognize the "compose" command. 

Thank you

**Reply by dansche** [reply]
No, I did not manage it to have the backup installing correctly, since than I have both Teslamates running and the new one records since installed, the old one I had on the raspberry still works and does record since beginning.. So still if there is someone experienced can help I would appreciate.

Hope we both get helped ;-)
